locales/en-US/notes.properties

```diff
@@ -1,12 +1,12 @@
-welcomeTitle=Welcome!
-welcomeText=This is a simple one-page notepad built in to Firefox that helps you get the most out of the Web.
+welcomeTitle2=Hello!
+welcomeText2=Welcome to this one-page notepad built in to Firefox. Browse the web. Jot down notes. It's that simple.
```

Comment:
# It’s nice to come back from PTO and see that, with respect to typographic punctuation, it’s handled correctly @flodolo.